charms/istio-pilot/src/charm.py

```diff
@@ -622,5 +636,34 @@ def _validate_upgrade_version(versions) -> bool:
     return True
 
 
+def _wait_for_update_rollout(
+    istioctl: Istioctl, retry_strategy: tenacity.Retrying, logger: logging.Logger
+):
+    for attempt in retry_strategy:
```

Comment:
how many times will it try until give up?

Reply:
just 5min?

Reply:
yeah 15 minutes

Reply:
from the code itself it is not that trivial to see. I mean "for attempt in retry", sounds like there will be several attempts. May be a short comment will be helpful

Reply:
every 2s for 15 minutes

Reply:
gotcha